package.json

```diff
@@ -65,5 +67,6 @@
 		"@types/throttle-debounce": "^5.0.0",
 		"@types/w3c-web-serial": "^1.0.3",
 		"vitest-canvas-mock": "^0.3.3"
-	}
+	},
+	"packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
what does this do?

Reply:
Again probably a side effect of my fixing errors. I will look into it.